tests/parsers/appbio_quantstudio/testdata/appbio_quantstudio_multiple_cal_doc_wells.json

```diff
@@ -495,6 +496,14 @@
                             {
                                 "data source identifier": "APPBIO_QUANTSTUDIO_TEST_ID_0",
                                 "data source feature": "cycle threshold result"
+                            },
```

Comment:
How are we "referencing" y-intercept and slope when they do not appear in the measurements? 

I see in the test files that there are columns for `y-intercept` and `slope` - shouldn't we add them to the processed data for this to make sense?

Reply:
Added check to add data sources only when their values are non None.

Reply:
@alejandro-salgado I'm still not clear that this is addressed. My point is not that the measurement could be None, my point is that the `y-intercept` and `slope` do not appear in the data source. There is no entry for `y-intercept` and `slope` in the measurement's processed data. I think you need to add them to the processed data?